srcpkgs/oneVPL/template

```diff
@@ -1,43 +1,28 @@
 # Template file for 'oneVPL'
 pkgname=oneVPL
 version=2023.3.1
-revision=2
-build_style=cmake
-hostmakedepends="pkg-config"
-makedepends="libva-devel libX11-devel"
+revision=3
+metapackage=yes
 short_desc="Intel oneAPI Video Processing Library"
 maintainer="Michael Aldridge <maldridge@voidlinux.org>"
-license="MIT"
-homepage="https://github.com/oneapi-src/oneVPL"
-changelog="https://github.com/oneapi-src/oneVPL/blob/master/CHANGELOG.md"
-distfiles="https://github.com/oneapi-src/oneVPL/archive/refs/tags/v$version.tar.gz"
-checksum=914530a80bdb155d3a4d50916ff34451da48584269e37075a48bd74a74e3d847
+license="Public Domain"
+homepage="https://github.com/intel/libvpl"
+changelog="https://github.com/intel/libvpl/blob/master/CHANGELOG.md"
 
-post_install() {
-	vlicense LICENSE
+oneVPL_package() {
+	metapackage=yes
+	short_desc" (transitional dummy package)"
+	depends="libvpl>=0"
 }
 
 oneVPL-devel_package() {
-	short_desc+=" - development files"
-	depends="${sourcepkg}>=${version}_${revision}"
-	pkg_install() {
-		vmove etc/vpl/vars.sh
-		vmove usr/include
-		vmove usr/lib/pkgconfig
-		vmove usr/lib/cmake
-		vmove "usr/lib/*.so"
-	}
+	metapackage=yes
+	short_desc" - development files"
```

Comment:
Btw, this isn't an assignment and it doesn't do anything. (e.g. the `short_desc` would still be `Intel oneAPI Video Processing Library`)
```suggestion
```

Reply:
went w/ none since that's your suggest line and it's transitional